ic_launcher.svg

```diff
@@ -1,9 +1 @@
-<?xml version="1.0" standalone="no"?>
-<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
-<svg width="216" height="216" viewBox="0 0 48 48" xmlns="http://www.w3.org/2000/svg" version="1.1">
-  <!--<rect width="48" height="48" fill="#283593" />-->
-  <path d="M44,24.2010101 L33.9004889,14.101499 L14.101499,33.9004889 L64.2010101,84 C49.2525804,63.9497929 54.2887564,61.9975027 58.1431296,58.1431296 C61.9975027,54.2887564 63.9497929,49.2525804 64,44.2010101 Z" fill="#1a237e" />
-  <path d="M24.0,24.0m-14.0,0.0a14.0,14.0,0,1,1,28.0,0.0a14.0,14.0,0,1,1,-28.0,0.0" fill="#5c6bc0" />
-  <path d="M37.7829445,26.469236 L29.6578482,18.3441397 L18.3441397,29.6578482 L26.469236,37.7829445 C29.1911841,37.2979273 31.7972024,36.0037754 33.9004889,33.9004889 C36.0037754,31.7972024 37.2979273,29.1911841 37.7829445,26.469236 Z" fill="#3f51b5" />
-  <path d="M24.0,24.0m-8.0,0.0a8.0,8.0,0,1,1,16.0,0.0a8.0,8.0,0,1,1,-16.0,0.0" fill="#FFFFFF" />
-</svg>
\ No newline at end of file
+<svg height="216" viewBox="0 0 48 48" width="216" xmlns="http://www.w3.org/2000/svg"><path d="m44 24.2010101-10.0995111-10.0995111-19.7989899 19.7989899 50.0995111 50.0995111c-14.9484297-20.0502071-9.9122537-22.0024973-6.0578805-25.8568704 3.8543731-3.8543732 5.8066633-8.8905492 5.8568704-13.9421195z" fill="#1a237e"/><path d="m10 24a14 14 0 1 1 28 0 14 14 0 1 1 -28 0" fill="#5c6bc0"/><path d="m37.7829445 26.469236-8.1250963-8.1250963-11.3137085 11.3137085 8.1250963 8.1250963c2.7219481-.4850172 5.3279664-1.7791691 7.4312529-3.8824556s3.3974384-4.7093048 3.8824556-7.4312529z" fill="#3f51b5"/><path d="m16 24a8 8 0 1 1 16 0 8 8 0 1 1 -16 0" fill="#fff"/></svg>
```

Comment:
The Screenshot_N.png files, art.png, and svg files compressed in this commit are not used and not packaged into the APK. art-min.png is an already-compressed and optimized png, which is used in README.md, thus this commit is removing readability from SVG files unnecessarily.

Reply:
Sorry about that. I have reverted the relevant images back to their original state